drivers/include/abp2.h

```diff
@@ -0,0 +1,283 @@
+/**
+ * @defgroup   drivers_abp2 ABP2 series Honeywell pressure and temperature sensor driver
+ * @ingroup    drivers_sensors
+ * @ingroup    drivers_saul
+ * @brief      I2C and SPI pressure and temperature sensor driver.
+ *
+ *
+ * ## Description
+ *
+ * The unit and the range depend on the part number of the sensor.
+ * It is the responsibility of the user to use homogeneous units, depending
+ * on the actual sensor being used.
+ * The pressure range is set when calling #abp2_init().
+ *
+ * This driver provides @ref drivers_saul capabilities.
+ *
+ * ## Usage
+ *
+ * See `RIOT/tests/drivers/abp2` for an example application using this driver.
+ *
+ * Add `USEMODULE += abp2` and `USEMODULE += abp2_spi` to the Makefile.
+ * When the I2C version is supported, the latter can be replaced by
+ * `USEMODULE += abp2_i2c`.
+ *
+ * When a measurement is started on the sensor, its internal ADC converts
+ * both pressure and temperature.
+ * These two values are retrieved in a single bus transfer.
+ *
+ * There are two ways to use the driver: either wait for the conversion to
+ * complete, or start a measurement while reading previous values.
+ * The first one is best suited for infrequent, non-periodic measurements,
+ * while the second one produces very compact code, but requires at least 5ms
+ * between two measurements.
+ * In any case, the raw values can be converted to
+ * physical quantities with #abp2_pressure() and #abp2_temperature().
+ *
+ * ### Poll sensor until the end of conversion
+ *
+ * -# Start a measurement with #abp2_measure().
+ * -# Poll the busy flag until it clears with #abp2_getstatus().
```

Comment:
This should be handled by the driver, the user should not need to care about such implementation details.

Reply:
OK. Added `abp2_read()`